editor/editor_node.h

```diff
@@ -868,7 +868,7 @@ class EditorNode : public Node {
 	Ref<Script> get_object_custom_type_base(const Object *p_object) const;
 	StringName get_object_custom_type_name(const Object *p_object) const;
 	Ref<Texture2D> get_object_icon(const Object *p_object, const String &p_fallback = "Object");
-	Ref<Texture2D> get_class_icon(const String &p_class, const String &p_fallback = "Object");
+	Ref<Texture2D> get_class_icon(const String &p_class, bool p_default_fallback = true, const String &p_fallback = "ArrowRight");
```

Comment:
Is this intentional? This shouldn't be changed this way, if so. If some tree/dialog needs this to be the default, then that's what it should pass to the method.

Reply:
So changing the default from Object to ArrowRight shouldn't be done, is that the issue you're mentioning? I don't remember why I'd changed it, probaby